### ISSUE TITLE
[Database] Charset

### ISSUE THREAD

**Issue by rogeriolino** [novosga/novosga#246]
Add charset option on database installation (use UTF-8 as default).

```php
// config/database.php
return array(
    // ... 
    "charset" => "utf8"
    // ...
);
```

**Comment by phansys** [comment]
Consider [```utf8mb4```](https://dev.mysql.com/doc/refman/5.5/en/charset-unicode-utf8mb4.html), available since MySQL 5.5.

**Comment by rogeriolino** [comment]
I've placed a input text to change it easily depending of the database choosed (default value is utf8).
